### ISSUE TITLE
Download stops midway when multiple pages from favorites are selected

### ISSUE THREAD

**Issue by xScarbo** [Xwilarg/NHentaiDownloader#54]
I have the latest release, and I download things as .zip and have the option 'download separately' ticked.
When I try to download my favorites in bulk, the process always stops midway. 

Initially what should've been 34 pages stopped after 17. Then 17-34 stopped at like page 25, and so on.
Is there a fix for this?

**Comment by xScarbo** [comment]
No, nothing on the console tab now, though I have finished downloading many hours ago.

**Comment by OldeDeus** [comment]
I encountered a similar issue on the latest firefox. It would freeze sometimes in the download UI box (doujinshi name/200.jpg please wait) and I dont know if its still working or not. This is occurring when I downloading from the favorites page
